### ISSUE TITLE
why do you keep doing this

### ISSUE THREAD

**Issue by typicalname0** [lighterlightbulb/SpaceMy#4]
i honestly dont understand what you get out of calling my actual spacemy "fake" and trying to claim mine steals your passwords. i dont understand why you just cant grow up for once and stop making your friends spam my repo with racial slurs and nsfw/gore issues/pull requests??? i literally havent done nothing to you and this is how you treat me, and i'm fine with you making a spacemy clone, but dont literally call it "spacemy" and call mine a fraud.

**Comment by typicalname0** [comment]
you've also been harrasing me for the past 2 years and sometimes it was justified but this really isnt, i don't understand what i did to you to deserve this, and i just want to be on neutral terms with you.

**Comment by typicalname0** [comment]
i can't stress this enough; and you guys have been stressing me out so lately. why do none of you guys have a sense of empathy?

**Comment by typicalname0** [comment]
...or are you going to keep ignoring me and continuing being an asshole

**Comment by typicalname0** [comment]
all i want is peace and for us to stop fighting

**Comment by johnfortnitekennedyCorp** [comment]
holy shit nigga STFU and get laid!

**Comment by lighterlightbulb** [comment]
> holy shit nigga STFU and get laid!

**Comment by thankyoutybg** [comment]
That's crazy but check out this awesome pig tho lol
![cool pig](https://user-images.githubusercontent.com/63983927/88979759-4c924780-d290-11ea-8961-633285e0ee84.gif)

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-gif-maker(2)](https://user-images.githubusercontent.com/68939829/88979983-1c9b7200-d299-11ea-81ce-ea376d9274c5.gif)

**Comment by johnfortnitekennedyCorp** [comment]
Get the fuck out of my porch

**Comment by lighterlightbulb** [comment]
Actually, I thought you'd deserve a proper response, so here goes:
- Recently, I have been told that you have been trying to contact my ISP to take down my internet for actions that I have never done. I have attempted to get in contact with you through the past, either through a surrogate or directly to no avail. You have blocked me on every place where I can contact you. I am being held accountable for actions that I have never done.
- You reported my GitHub profile for something I never did.
- Even before, in the months of January through May 2020 you have been asking numerous people for help on writing a manifesto on why I should be denied internet access. 

This repository is still going to be up, and I will still continue active development on this. I did not "force" my friends to spam your repository. They did it out of their free will.
 
I have not been ignoring you, I have just been sent a link to this issue.

For you to say that I lack a sense of empathy could not be more ironic as you go and pose as this good Samaritan whenever you talk to me and then go huddle up in your Discord and then shit talk me. I could care less about the shit talking; but among the shittalking, you then have tried to set up an IP grabber to "catch" me, contacted my friends in an attempt to sever my friendship with them, attempted to DOX me, tried creating an exposé of me for stuff I have never done, among other stuff. It is interesting how much you care about me. If you so wish, I am single :-)

I'd write a more concise response but you'd ignore all of it and label me as "mentally unstable" as you have done so in the past.

In short, stay mad

**Comment by johnfortnitekennedyCorp** [comment]
lol

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(3)](https://user-images.githubusercontent.com/68939829/88980684-a0a22980-d29a-11ea-989d-3104d261e905.gif)

**Comment by lighterlightbulb** [comment]
Copy of report send to GitHub.com regarding @typicalname0:

```
Hi there,

My name is lightbulblighter, an aspiring young developer. Recently, I have been notified that a user by the name of typicalname0 has been open-sourcing my old copy of code (seen here: https://github.com/typicalname0/spacemy.xyz). I am fine with this; this is not the reason I am reporting typicalname0. However, he is using GitHub to lie and slander my name. For one, he is using the GitHub platform to host an abusive phishing site (https://spacemy.xyz, the website source is different than the one on the GitHub repository) and then further claiming that *his* version is the official one. This is a lie. 

You can see the commit history for his README and the clearly copy-pasted notice from my README only a few hours later.
copy: https://github.com/typicalname0/spacemy.xyz/commits/master/README.md
original: https://github.com/lighterlightbulb/SpaceMy/commits/master/README.md (compare commit timestamps)

I love open source, and GitHub makes it very easy for me to continue my ventures into the open source community. It is toxic people like this that drive others away from ever touching open source, though.

To summarize, typicalname0 is using GitHub to slander my name, phish users, and submit false reports about me, claiming I had done something with extreme scenarios. I would like for the `spacemy.xyz` repository to be taken down, as I do not believe that he is a bad person, but I don't want my users to be deceived and tricked into giving their passwords away.

Best regards,
lightbulblighter```

![firefox_Ycmr3y9K5o](https://user-images.githubusercontent.com/59720715/88981016-9aa44c80-d28a-11ea-80e1-35a11146108b.png)

**Comment by johnfortnitekennedyCorp** [comment]
LOL

**Comment by ryelow90210** [comment]
Shouldve just my pull request #2

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(4)](https://user-images.githubusercontent.com/68939829/88981271-f6c39c80-d29b-11ea-9792-004964163e32.gif)

**Comment by exieln** [comment]
Bazinga

**Comment by johnfortnitekennedyCorp** [comment]
Bazinga

**Comment by johnfortnitekennedyCorp** [comment]
porch

**Comment by lighterlightbulb** [comment]
Bajomama

**Comment by johnfortnitekennedyCorp** [comment]
@ @ @ @

**Comment by typicalname0** [comment]
```
For you to say that I lack a sense of empathy could not be more ironic as you go and pose as this good Samaritan whenever you talk to me and then go huddle up in your Discord and then shit talk me. I could care less about the shit talking; but among the shittalking, you then have tried to set up an IP grabber to "catch" me, contacted my friends in an attempt to sever my friendship with them, attempted to DOX me, tried creating an exposé of me for stuff I have never done, among other wacky things. It is interesting how much you care about me. If you so wish, I am single :-)```

i didnt try to attempt to dox you??? all i did was ask if someone had carrot's ip (which i will admit was wrong on my part) so i could email your isp that you have my dox, etc etc. i tried to ""expose"" you but that was a dumb idea so i didnt do it, and i didnt think it was a good idea or a smart idea to "expose" you. i will admit i have done some stupid things that i shouldn't have and i'm sorry for that

**Comment by typicalname0** [comment]
all i want to do is be on equal terms and for both of us to calm down

**Comment by wertyc** [comment]
lighter you faggot

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(3)](https://user-images.githubusercontent.com/68939829/88986416-48265880-d2a9-11ea-8784-ad9b960398ea.gif)

**Comment by lighterlightbulb** [comment]
> lighter you faggot

No bad words

**Comment by exieln** [comment]
![beast](https://user-images.githubusercontent.com/25045482/88986514-33de5d80-d2a1-11ea-975c-c19680afba40.png)
@typicalname0 this you?

**Comment by johnfortnitekennedyCorp** [comment]
LOL

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(3)](https://user-images.githubusercontent.com/68939829/88986598-d7337080-d2a9-11ea-931c-6c7a0b65d1bc.gif)

**Comment by lighterlightbulb** [comment]
@typicalname0 Dis u?
![unknow1n](https://user-images.githubusercontent.com/59720715/88986740-77cd6480-d299-11ea-9acf-4584b666e492.png)
![unknown-115](https://user-images.githubusercontent.com/59720715/88986742-7865fb00-d299-11ea-92ea-3c1417009dfd.png)
![unknown-224](https://user-images.githubusercontent.com/59720715/88986745-7865fb00-d299-11ea-9e06-1ba37293d23d.png)
![unknown-117](https://user-images.githubusercontent.com/59720715/88986746-7865fb00-d299-11ea-8a74-42902d00a3b2.png)
![unknown-73](https://user-images.githubusercontent.com/59720715/88986747-7865fb00-d299-11ea-8e6a-a532967c938a.png)
![unknown-116](https://user-images.githubusercontent.com/59720715/88986749-7865fb00-d299-11ea-9837-7a9c0495b390.png)
![unknown-86](https://user-images.githubusercontent.com/59720715/88986750-78fe9180-d299-11ea-8517-0be105cf0070.png)
![unknown-122](https://user-images.githubusercontent.com/59720715/88986751-78fe9180-d299-11ea-8e64-cbdead6b3080.png)
![unknown-77](https://user-images.githubusercontent.com/59720715/88986752-78fe9180-d299-11ea-9ba3-47474d0e10d4.png)
![unknown-171](https://user-images.githubusercontent.com/59720715/88986754-78fe9180-d299-11ea-85f2-38e51ebe88f6.png)
![unknown-220](https://user-images.githubusercontent.com/59720715/88986755-78fe9180-d299-11ea-9d02-d7804c668916.png)
![unknown-99](https://user-images.githubusercontent.com/59720715/88986756-79972800-d299-11ea-8293-e340563c9018.png)
![unknown-170](https://user-images.githubusercontent.com/59720715/88986757-79972800-d299-11ea-9c5d-4ced85baabf7.png)
![unknown-83](https://user-images.githubusercontent.com/59720715/88986758-79972800-d299-11ea-98ef-2ebc71a6e79c.png)
![unknown-85](https://user-images.githubusercontent.com/59720715/88986759-7a2fbe80-d299-11ea-9e93-7840d243a1ee.png)

**Comment by lighterlightbulb** [comment]
> all i want to do is be on equal terms and for both of us to calm down

LOL
prime example of this: "as you go and pose as this good Samaritan whenever you talk to me and then go huddle up in your Discord and then shit talk me."
![88986514-33de5d80-d2a1-11ea-975c-c19680afba40](https://user-images.githubusercontent.com/59720715/88986822-a9dec680-d299-11ea-9097-089ff7b26646.png)

**Comment by lighterlightbulb** [comment]
OFFICIAL SpaceMy v1.0.0 Coming out soon!

**Comment by typicalname0** [comment]
carrot it's the fact that you're claiming mine steals your fucking password and that mine is a fraud + you just wont shut the fuck up for 1 day

**Comment by lighterlightbulb** [comment]
okay but yours stole my password i registered and my keyboard blew up. TypicalName's SpaceMy is a RAT and I've notified the FBI

**Comment by lighterlightbulb** [comment]
spacemy is a glorified ip logger 💅

**Comment by typicalname0** [comment]
i am hacking your computer right now. [0%]

**Comment by typicalname0** [comment]
[2%]

**Comment by typicalname0** [comment]
HACKING!!!!!!!!!!!

**Comment by lighterlightbulb** [comment]
Bro at what point are you gonna stop sperging LOL

**Comment by typicalname0** [comment]
[90%] Almost complete im going to blow up your keyboard

**Comment by lighterlightbulb** [comment]
i honestly dont understand what you get out of calling my actual spacemy "fake" and trying to claim mine steals your passwords. i dont understand why you just cant grow up for once and stop making your friends spam my repo with racial slurs and nsfw/gore issues/pull requests??? i literally havent done nothing to you and this is how you treat me, and i'm fine with you making a spacemy clone, but dont literally call it "spacemy" and call mine a fraud.

**Comment by typicalname0** [comment]
and how the hell is mine a fucking ip logger when we clearly dont store ip's on databases and the source code is on github

**Comment by ryelow90210** [comment]
![image](https://user-images.githubusercontent.com/66385413/88987176-21fdba00-d2a3-11ea-9977-b058e159b743.png)

**Comment by typicalname0** [comment]
what

**Comment by ryelow90210** [comment]
You wouldnt get it... Political Humor!!!!

**Comment by typicalname0** [comment]
I am running for president i am a politician

**Comment by typicalname0** [comment]
kanye 2020

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(3)](https://user-images.githubusercontent.com/68939829/88987248-b53aed80-d2ab-11ea-9afe-38644fd059aa.gif)

**Comment by typicalname0** [comment]
why are you at my front porch

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-video-to-gif(4)](https://user-images.githubusercontent.com/68939829/88987272-cb48ae00-d2ab-11ea-9537-1c504bb26fc2.gif)

**Comment by typicalname0** [comment]
how are you alive i thought you were dead jfk

**Comment by ryelow90210** [comment]
typical is like "Im so dumb im so dumb" because he cant set up  **_**Github Actions

**Comment by typicalname0** [comment]
Uhhhm yeah i can setup **__**Github Actions

**Comment by lighterlightbulb** [comment]
> and how the hell is mine a fucking ip logger when we clearly dont store ip's on databases and the source code is on github

idk the source code from the github is different o site because you didn't commit the ip logger code

**Comment by typicalname0** [comment]
why would i log ips

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-gif-maker(2)](https://user-images.githubusercontent.com/68939829/88987362-106ce000-d2ac-11ea-82c3-d050a6cbd248.gif)

**Comment by johnfortnitekennedyCorp** [comment]
bc ur a drama queen

**Comment by typicalname0** [comment]
u cant do anything with an ip

**Comment by typicalname0** [comment]
ok that's not true

**Comment by johnfortnitekennedyCorp** [comment]
lol ok typical

**Comment by ryelow90210** [comment]
you can do yo mama with an ip L O L

**Comment by typicalname0** [comment]
who even are you JFK

**Comment by ryelow90210** [comment]
john f kennedy
the president

**Comment by johnfortnitekennedyCorp** [comment]
the fbi agent

**Comment by johnfortnitekennedyCorp** [comment]
yea

**Comment by typicalname0** [comment]
i thought you died

**Comment by johnfortnitekennedyCorp** [comment]
i thought you stopped being a drama queen

**Comment by typicalname0** [comment]
you were the guys to start doing this shit and make the repo

**Comment by typicalname0** [comment]
i didnt say anything and then once i join carrot's roblox client archive server you guys go swarming all over me

**Comment by typicalname0** [comment]
what do you get out of this and what is the point

**Comment by typicalname0** [comment]
?

**Comment by typicalname0** [comment]
hellloooooooooooooo

**Comment by lighterlightbulb** [comment]
I'm eating food rn babe I'll respond later but just know that there will be some funny repercussions coming soon

**Comment by johnfortnitekennedyCorp** [comment]
![tenor](https://user-images.githubusercontent.com/68939829/88987548-ab65ba00-d2ac-11ea-9459-22dbf0a6620c.gif)

**Comment by typicalname0** [comment]
what are "funny repercussions"

**Comment by lighterlightbulb** [comment]
like a dmca

**Comment by typicalname0** [comment]
dmca for what....

**Comment by lighterlightbulb** [comment]
I forgot the specifics but you'll know when you see em. I'm not well versed in legal stuff

**Comment by typicalname0** [comment]
so you're going to false dmca me????

**Comment by johnfortnitekennedyCorp** [comment]
WOW NO ONE CARES

**Comment by typicalname0** [comment]
????

**Comment by typicalname0** [comment]
if this is going to the point of where i'm going to be false dmca'd i'll just shut this down. i'll do anything

**Comment by johnfortnitekennedyCorp** [comment]
![ezgif com-optimize(3)](https://user-images.githubusercontent.com/68939829/88987906-c2f17280-d2ad-11ea-8a45-073d1b4cf2dd.gif)

**Comment by lighterlightbulb** [comment]
Lmao do it if you want but they're going to after you individually.Youd have to clear your internet name idk. The process has already been set in motion by maff

**Comment by lighterlightbulb** [comment]
you'd have to ask maff about most of this, not my place

**Comment by typicalname0** [comment]
Lmao!!!!!! they're going to after you individually and going to rob ur house.Youd have to clear your internet name idk. The process has already been set in motion by my hacker brother joe.

**Comment by lighterlightbulb** [comment]
LOL

**Comment by lighterlightbulb** [comment]
Dude i'mma take this down too so

**Comment by johnfortnitekennedyCorp** [comment]
You can't perform that action at this time

**Comment by johnfortnitekennedyCorp** [comment]
Not cool man!

**Comment by typicalname0** [comment]
you'd have to ask my hacker brother joe, he watches hacking with ramzi and hacks roblox. not my place

**Comment by lighterlightbulb** [comment]
well anyway I wish ya luck typical. I don't think you're a bad guy and I don't hate ya but some of your actions are in bad taste.
I'll take down this repo tomorrow in response to the C&D And I advise you to do so as well. I don't like legal troubles.
The most interesting takeaway from all of this is how easily the orc, or for the matter, spacemy "community" can hate someone. Like bro... it's internet shit! But somehow they still manage to hate
I can't wrap my head the reason why people choose to hate rather than dislike, but iunno.

Cya

**Comment by johnfortnitekennedyCorp** [comment]
Holy deez 89 comments

**Comment by lighterlightbulb** [comment]
Blocking typicalnae,
